traits/has_traits.py

```diff
@@ -385,6 +394,11 @@ def update_traits_class_dict(class_name, bases, class_dict):
     prefix_list = []
     view_elements = {}
 
+    # Mapping from method name or trait name to list(dict)
```

Comment:
I am sorry for the confusion - the keys here are only method names.
I went ahead with the `Property` support in the other branches where the `name` (defined in the class dict) could refer to the name of a trait that is a property. Here we are not dealing with property so that half of the comment is misleading.  I will remove that.